tools/construct_npy.cpp

```diff
@@ -43,8 +43,7 @@ void run(float *data,
             << std::endl;
 
```

Comment:
Calling the template function "run" is a really, _really_ nice solution to the tooling problem where it's hard to have easy c++ tools because the index is templated on the DistanceInterface. Nice one!